certified-connectors/USB4SAP/apidefinition.swagger.json

```diff
@@ -0,0 +1,70 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "USB4SAP",
+    "version": "1.0",
+    "description": "Universal Synch Bridge for SAP: One API for all SAP integration, analytics and UI needs. Simplify modern analytics, apps and automations. Provision SAP data, insights and automation On-demand.",
+    "contact":{
+      "name": "Ecoservity",
+      "url" : "https://www.ecoservity.com/#contact",
+      "email": "hello@ecoservity.com"
+
+    }
+  },
+
+  "host" : "http://35.208.99.120:8000",
```

Comment:
@ecsadm the host must be a valid URI for production environment. Test or staging environments are not allowed.